RELEASE_NOTES.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## 3.0.0 (2016-02-07)
+
+#### BREAKING CHANGES
+* `git hack`: no longer accepts a parent branch
```

Comment:
can you also include a reference to the issue so users can see the discussion and rationale for the change?